eva_submission/ingestion_templates.py

```diff
@@ -0,0 +1,119 @@
+def accession_props_template(
+        instance_id,
+        assembly_accession,
+        taxonomy_id,
+        project_accession,
+        vcf_path,
+        aggregation,
+        fasta,
+        output_vcf,
+        report,
+        postgres_url,
+        postgres_user,
+        postgres_pass,
+        mongo_host,
+        mongo_user,
+        mongo_pass,
+):
+    return f"""
+accessioning.instanceId=instance-{instance_id}
+accessioning.submitted.categoryId=ss
+accessioning.monotonic.ss.blockSize=100000
+accessioning.monotonic.ss.blockStartValue=5000000000
+accessioning.monotonic.ss.nextBlockInterval=1000000000
+
+parameters.assemblyAccession={assembly_accession}
+parameters.taxonomyAccession={taxonomy_id}
+parameters.projectAccession={project_accession}
+parameters.chunkSize=100
+parameters.vcf={vcf_path}
+parameters.vcfAggregation={aggregation}
+parameters.forceRestart=false
+parameters.fasta={fasta}
+parameters.outputVcf={output_vcf}
+parameters.assemblyReportUrl=file:{report}
+# contigNaming available values: SEQUENCE_NAME, ASSIGNED_MOLECULE, INSDC, REFSEQ, UCSC, NO_REPLACEMENT
+parameters.contigNaming=NO_REPLACEMENT
+
+spring.batch.job.names=CREATE_SUBSNP_ACCESSION_JOB
+
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url={postgres_url}
+spring.datasource.username={postgres_user}
+spring.datasource.password={postgres_pass}
+spring.datasource.tomcat.max-active=3
+spring.jpa.generate-ddl=true
+
+spring.data.mongodb.host={mongo_host}
+spring.data.mongodb.port=27017
+spring.data.mongodb.database=eva_accession_sharded
+spring.data.mongodb.username={mongo_user}
+spring.data.mongodb.password={mongo_pass}
+spring.data.mongodb.authentication-database=admin
+mongodb.read-preference=primaryPreferred
+
+spring.main.web-environment=false
+"""
+
+
+def variant_load_props_template(
+        project_accession,
+        analysis_accession,
+        vcf_path,
+        aggregation,
+        study_name,
+        fasta,
+        db_name,
+        vep_species,
+        vep_version,
+        vep_cache_version,
+        annotation_skip=False,
+):
+    return f"""
+# JOB
+spring.batch.job.names=genotyped-vcf-job
+
+# SUBMISSION FIELDS
+input.study.id={project_accession}
+input.vcf.id={analysis_accession}
+input.vcf={vcf_path}
+input.vcf.aggregation={aggregation}
+
+input.study.name={study_name}
+input.study.type=COLLECTION
+
+input.pedigree=
+input.fasta={fasta}
+
+output.dir=/nfs/production3/eva/data/{project_accession}/40_transformed/
+output.dir.annotation=/nfs/production3/eva/data/{project_accession}/51_annotation/
+output.dir.statistics=/nfs/production3/eva/data/{project_accession}/50_stats/
```

Comment:
We should avoid having path from the cluster in public github. Get the path from the config file.